dotnet/src/SemanticKernel.Abstractions/SemanticKernel.Abstractions.csproj

```diff
@@ -18,6 +18,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
     <PackageReference Include="Microsoft.Bcl.HashCode" />
+    <PackageReference Include="Microsoft.CSharp" />
```

Comment:
Why is this needed?

Reply:
Good catch, that was for a different option. Will remove this.